MIT.md

```diff
@@ -1,10 +1,7 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [Bhekisile & Akorede]
 
```

Comment:
- [ ] Kindly correct the year in your license file 👍

Reply:
Fixed. Thank you